lib/datadog/tracing/configuration/settings.rb

```diff
@@ -277,6 +277,22 @@ def self.extended(base)
                   o.default { env_to_float(Tracing::Configuration::Ext::Sampling::ENV_RATE_LIMIT, 100) }
                 end
 
+                # Trace sampling rules.
+                # These rules control whether a trace is kept or dropped by the tracer.
+                #
+                # The `rules` format is a String with a JSON array of objects:
+                # Each object must have a `sample_rate`, and the `name` and `service` fields
+                # are optional. The `sample_rate` value must be between 0.0 and 1.0 (inclusive).
+                # `name` and `service` are Strings that allow the `sample_rate` to be applied only
+                # to traces matching the `name` and `service`.
+                #
+                # @default `DD_TRACE_SAMPLING_RULES` environment variable. Otherwise `nil`.
+                # @return [String,nil]
+                # @public_api
+                option :rules do |o|
+                  o.default { ENV.fetch(Configuration::Ext::Sampling::ENV_RULES, nil) }
```

Comment:
Isn't `ENV.fetch(Configuration::Ext::Sampling::ENV_RULES, nil)` the same as `ENV[Configuration::Ext::Sampling::ENV_RULES]`?

Reply:
Yeah, we just write `ENV.fetch(blah, nil)` more than `ENV#[]` in this file.

I feel like this will mostly go away when we refactor configuration 😉

Reply:
I'll leave this one like this, because it will be simple to refactor when this code gets simplified in the configuration refactoring.